pkg/config/pronom.go

```diff
@@ -202,8 +203,17 @@ func TextPuid() string {
 // SetDroid sets the name and/or location of the DROID signature file.
 // I.e. can provide a full path or a filename relative to the HOME directory.
 func SetDroid(d string) func() private {
+	pronom.droid = d
```

Comment:
pronom.droid should be set within the closure not outside it. This means that the option will only take effect when the identifer is made. Making this change will mean that the option gets immediately set (changing behaviour of code in cmd/roy).  Generally if you need config options like this to take immediate effect you can do so by invoking like this: SetDroid("x")() [that second parens calls the returned option func immediately]

Reply:
Ah! That explains why it wasn't working - will give this a try.